src/clausetable.jl

```diff
@@ -0,0 +1,115 @@
+
+"Nested dictionary to store indexed clauses."
+const ClauseTable = Dict{Symbol,Dict{Symbol,Vector{<:AbstractClause}}}
```

Comment:
I like this. One thing that is unclear to me is whether w would be able to mix different subtypes of `AbstractClause` within the same `ClauseTable`?
This should be a desired case because Problog programs will also contain a lot of Horn Clauses.

Reply:
Yes, that would be the case. Will change it to your suggestion.